#! /bin/sh /usr/share/dpatch/dpatch-run
## 17_man__mysqld__section.dpatch by  <ch@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: MySQL bug #21220

@DPATCH@

#
# File will be moved to mysqld.8 as mysqld is in /usr/sbin
#
--- old/man/mysqld.1.orig	2004-06-01 01:03:50.000000000 +0200
+++ new/man/mysqld.1	2004-06-01 01:03:56.000000000 +0200
@@ -1,2 +1,2 @@
-.TH "\\FBMYSQLD\\FR" "1" "07/28/2006" "MySQL 5.0" "MySQL Database System"
+.TH "\\FBMYSQLD\\FR" "8" "07/28/2006" "MySQL 5.0" "MySQL Database System"
 .SH NAME
